### DISCUSSION TITLE
cadvisor podman integration

### DISCUSSION THREAD

**Discussion by towe75** [containers/podman#9171] [Show and tell]
Hey,

I wanted to let you know that im working on podman cadvisor support. PR is at https://github.com/google/cadvisor/pull/2794
It's for now based on the docker compatibility API and a shameless clone of the cadvisor docker code. 

Let's see if people like it and PR gets approved. I consider porting it to pkg/binding and native libpod API to get a filesystem watcher and later also rootless support if my spare time allows it.

**Comment by rhatdan** [comment]
Cool, thanks.
